db/seeds.rb

```diff
@@ -8,6 +8,7 @@
                     fetch(Rails.env, {})
 
 service_providers.each do |issuer, config|
+  next if Figaro.env.chef_env == 'prod' && config['live'] != 'true'
```

Comment:
Good point @zachmargolis 

unrelated questions:
* why do we care about "live" status for non-prod envs?
* if we do not want to create these SPs for "real" prod? If not, how will they be created?